util/buildflags/attests.go

```diff
@@ -159,6 +159,8 @@ func (a *Attest) UnmarshalText(text []byte) error {
 				return errors.Wrapf(err, "invalid value %s", field)
 			}
 			a.Disabled = disabled
+		case "generator":
```

Comment:
Why is there special handling for "generator" key in here?

Reply:
I believe that key is sensitive in BuildKit

https://github.com/moby/buildkit/blob/824bc35b9771357e548ca913f8790854b70cf1f5/control/control.go#L479

Reply:
Yeah, if I modify these tests here:

https://github.com/moby/buildkit/blob/824bc35b9771357e548ca913f8790854b70cf1f5/frontend/attestations/parse_test.go#L19

and add 

```go
{
	name: "test",
	values: map[string]string{
		"attest:sbom": "GENERATOR=docker.io/foo/bar",
	},
	expected: map[string]map[string]string{
		"sbom": {
			"GENERATOR": "docker.io/foo/bar",
		},
	},
},
```

I get

```
 Error:          Not equal:
                                expected: map[string]map[string]string{"sbom":map[string]string{"GENERATOR":"docker.io/foo/bar"}}
                                actual  : map[string]map[string]string{"sbom":map[string]string{"GENERATOR":"docker.io/foo/bar", "generator":"docker/buildkit-syft-scanner:stable-1"}}
```

Reply:
Yeah, you're right. It's not a big impact